WordPress/src/main/java/org/wordpress/android/ui/domains/DomainRegistrationDetailsFragment.kt

```diff
@@ -519,4 +520,12 @@ class DomainRegistrationDetailsFragment : Fragment() {
             AndroidSupportInjection.inject(this)
         }
     }
+
+    override fun onResume() {
+        super.onResume()
+        if (activity is ScrollableViewInitializedListener) {
+            (activity as ScrollableViewInitializedListener)
```

Comment:
you could do something like `(activity as? ScrollableViewInitializedListener)?.onScrollableViewInitialized` which is a bit shorter :-)

Reply:
Nice! Thanks for the tip :)